<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageIden</class>
 <widget class="QMainWindow" name="ImageIden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>图像识别</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>250</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>471</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>舵机控制</string>
    </property>
    <widget class="QPushButton" name="btUp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ahead</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btRight">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btDown">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>竖直控制</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>水平控制</string>
     </property>
    </widget>
    <widget class="QDial" name="horiCtlDial">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="vertCtlSlider">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>21</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="horiValueSBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="vertValueSBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>图片存储</string>
    </property>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>名字</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>延时</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btSave">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>连拍</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btPhoto">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>抓拍</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numberSBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="delaySBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>263</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="grayBox">
       <property name="text">
        <string>灰度图显示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbRefrashImg">
       <property name="text">
        <string>定时刷新图像</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="saveProgressBar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btPrePic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>101</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btNextPic">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>101</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionFromFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionIden">
   <property name="text">
    <string>识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionZuoqu">
   <property name="text">
    <string>zhuqu</string>
   </property>
  </action>
  <action name="actionXunliang">
   <property name="text">
    <string>xunliang</string>
   </property>
  </action>
  <action name="actionFromFile">
   <property name="text">
    <string>载入图片</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionCatch">
   <property name="text">
    <string>捕获</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>训练</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人脸识别</string>
   </property>
   <property name="iconText">
    <string>人脸识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionDigit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>数字识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionAa">
   <property name="text">
    <string>aa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
